k8_cortx_cloud/upgrade-cortx-cloud.sh

```diff
@@ -318,9 +328,29 @@ case "${UPGRADE_TYPE}" in
         ;;
 esac
 
+cortx_deployments=()
 # Wait for all CORTX Pods to be ready
+while IFS= read -r line; do
+    IFS=" " read -r -a deployments <<< "${line}"
+    cortx_deployments+=("deployment/${deployments[0]}")
+done < <(kubectl get deployments --namespace="${namespace}" | grep "${cortx_deployment_filter}")
+sleep "${TIMEDELAY}";
 printf "\nWait for CORTX Pods to be ready"
-wait_for_cortx_pods
+if ! wait_for_all_deployments_available 300s "CORTX PODs" "${cortx_deployments[@]}"; then
```

Comment:
Please change the name to generic one i.e. "wait_for_all_pods_available", so that we mimnmize the rework when we switch to statefulsets.